db/migrate/20231010153003_create_participant_id_changes.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class CreateParticipantIdChanges < ActiveRecord::Migration[7.0]
+  def change
+    create_table :participant_id_changes do |t|
+      t.references :user, null: false, foreign_key: true
+
+      t.references :from_participant, null: false, type: :uuid
```

Comment:
maybe we can name those from user id/to user id 🤔 ?

Reply:
@cwrw 
I went back and forth with this lol
I didn't want the API to be "participant_id_changes" and the table to be "user_id_changes", thought might be good to stay consistent.